README.yaml

```
@@ -75,20 +95,61 @@ usage: |-
       plan:
         runs-on: ubuntu-latest
         steps:
-          - name: Plan Atmos Component
-            uses: cloudposse/github-action-atmos-terraform-apply@v1
+          - name: Terraform Apply
+            uses: cloudposse/github-action-atmos-terraform-apply@v2
             with:
               component: "foobar"
               stack: "plat-ue2-sandbox"
-              component-path: "components/terraform/s3-bucket"
-              terraform-apply-role: "arn:aws:iam::111111111111:role/acme-core-gbl-identity-gitops"
-              terraform-state-bucket: "acme-core-ue2-auto-gitops"
-              terraform-state-role: "arn:aws:iam::999999999999:role/acme-core-ue2-auto-gitops-gha"
-              terraform-state-table: "acme-core-ue2-auto-gitops"
-              aws-region: "us-east-2"
+  ```
 
+  ### Migrating from `v1` to `v2`
+  
+  `v2` drop `component-path` variable. Now it fetches from `atmos.yaml` file automatically.
+  
+  `v2` moved variables from `inputs` to atmos gitops config path `./.github/config/atmos-gitops.yaml`
+  
```

Here are the major changes migrating from `v1` to `v2`
```suggestion
1.   `v2` drops the `component-path` variable and instead fetches if directly from the [`atmos.yaml` file](https://atmos.tools/cli/configuration/) automatically. Simply remove the `component-path` argument from your invocations of the `cloudposse/github-action-atmos-terraform-apply` action.
2.  `v2` moves most of the `inputs` to the Atmos GitOps config path `./.github/config/atmos-gitops.yaml`. Simply create this file, transfer your settings to it, then remove the corresponding arguments from your invocations of the `cloudposse/github-action-atmos-terraform-apply` action.
```